### DISCUSSION TITLE
Internal visibility in library

### DISCUSSION THREAD

**Discussion by Tarreef** [Cyfrin/foundry-full-course-cu#2808] [Q&A]
Pleas why is the visibility type in a library set to "internal"...
and what happens if it's without it or it's not indicated. will the contract fail to deploy??

**Comment by EngrPips** [comment]
Hello @Tarreef, This https://medium.com/@ajaotosinserah/mastering-libraries-in-solidity-36d783409dfc  will satisfy your curiosity.
